config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/config/loader/UnsupportedEntityLoader.java

```diff
@@ -0,0 +1,100 @@
+package com.ca.apim.gateway.cagatewayconfig.config.loader;
+
+import com.ca.apim.gateway.cagatewayconfig.beans.Bundle;
+import com.ca.apim.gateway.cagatewayconfig.beans.EntityUtils;
+import com.ca.apim.gateway.cagatewayconfig.beans.UnsupportedGatewayEntity;
+import com.ca.apim.gateway.cagatewayconfig.util.IdGenerator;
+import com.ca.apim.gateway.cagatewayconfig.util.json.JsonTools;
+import com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentParseException;
+import com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentTools;
+import com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils;
+import org.jetbrains.annotations.NotNull;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.io.File;
+import java.util.List;
+import java.util.Map;
+
+import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.*;
+import static com.ca.apim.gateway.cagatewayconfig.util.gateway.BundleElementNames.*;
+
+@Singleton
+public class UnsupportedEntityLoader extends EntityLoaderBase<UnsupportedGatewayEntity> implements EntityLoader {
+
+    private final EntityUtils.GatewayEntityInfo gatewayEntityInfo;
+
+    @Inject
+    public UnsupportedEntityLoader(final JsonTools jsonTools, final IdGenerator idGenerator) {
+        super(jsonTools, idGenerator);
+        this.gatewayEntityInfo = EntityUtils.createEntityInfo(UnsupportedGatewayEntity.class);
+    }
+
+    @Override
+    public void load(final Bundle bundle, final File rootDir) {
```

Comment:
@uppoju , do we need this load method, env entities loads with below load method, also here folder is hard coded to config directory

Reply:
Unsupported entities have xml along with yml file, super.load methods loads only yml file
Even for other environment entities jsonTools.getDocumentFile(rootDir, getFileName()) loads from config folder only